test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -36,6 +36,33 @@ public void elementsAreUnique() throws Exception {
         assertNotUnique(null, "a", "b", null);
     }
 
+    @Test
+    public void isAnyNull() throws Exception {
+        // objects with no null
+        assertIsNotNull(1);
+        assertIsNotNull("abc");
+        assertIsNotNull(Integer.valueOf(1));
+
+        // objects with one null
+        assertIsNull((Object) null);
+        assertIsNull(1, null);
+        assertIsNull("abc", null);
+        assertIsNull(Integer.valueOf(1), null);
+
+        // objects with more than one null
+        assertIsNull(null, null, null);
+        assertIsNull(null, 1, null);
+        assertIsNull(null, "abc", null);
+        assertIsNull(Integer.valueOf(1), null, null);
+    }
+
+    private void assertIsNull(Object... objects) {
```

Comment:
It's good that you followed the style used by the code nearby.
However, in this case, there is no need to abstract the assertTrue(Utils.isAnyNull(...)) as a separate method as it doesn't add much value.
The existing test uses assertAreUnique() in order to prevent Arrays.asList(..) from being used each time.